From e56a2e9f805b7d62c187ac5791c75e0a162ac82b Mon Sep 17 00:00:00 2001
From: Alicja Michalska <ahplka19@gmail.com>
Date: Thu, 26 Oct 2023 20:54:31 +0200
Subject: [PATCH 02/11] arm64: dts: mediatek: mt8183: Mali OPP tables...

Ammendum: It is now managed by mtk-regulator-coupler module.
Introduced in upstream commit: c200774a6df47cfc1053334c98ab777563de5bd8.

Signed-off-by: Alicja Michalska <ahplka19@gmail.com>
---
 arch/arm64/boot/dts/mediatek/mt8183.dtsi | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm64/boot/dts/mediatek/mt8183.dtsi b/arch/arm64/boot/dts/mediatek/mt8183.dtsi
index 1aec57314ea5..e0c4c2357a4e 100644
--- a/arch/arm64/boot/dts/mediatek/mt8183.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt8183.dtsi
@@ -561,11 +561,9 @@ gpu_opp_table: opp-table-0 {
 		compatible = "operating-points-v2";
 		opp-shared;
 
-<<<<<<< HEAD
 		opp-300000000 {
 			opp-hz = /bits/ 64 <300000000>;
 			opp-microvolt = <625000>;
-=======
 			opp-300000000 {
 				opp-hz = /bits/ 64 <300000000>;
 				opp-microvolt = <625000>;
@@ -661,7 +659,6 @@ opp-800000000 {
 				opp-microvolt = <825000>;
 				opp-core-mask = /bits/ 64 <0xf>;
 			};
->>>>>>> 1909ff276244 (arm64: dts: mediatek: mt8183: Mali OPP tables...)
 		};
 
 		opp-320000000 {
-- 
2.41.0

